File: 2026POTD/JANUARY/25-01-2026/1984.py
class Solution:
    def minimumDifference(self, nums, k):
        if k <= 1:
            return 0
        nums.sort()
        ans = float('inf')
        for i in range(len(nums) - k + 1):
            ans = min(ans, nums[i + k - 1] - nums[i])
        return ans

File: 2026POTD/JANUARY/25-01-2026/1984.rs
impl Solution {
    pub fn minimum_difference(mut nums: Vec<i32>, k: i32) -> i32 {
        if k <= 1 {
            return 0;
        }
        nums.sort();
        let k = k as usize;
        let mut ans = i32::MAX;
        for i in 0..=nums.len() - k {
            ans = ans.min(nums[i + k - 1] - nums[i]);
        }
        ans
    }
}